spinalcordtoolbox/scripts/sct_merge_images.py

```diff
@@ -42,7 +42,23 @@ def get_parser():
     # Initialize the parser
 
     parser = SCTArgumentParser(
-        description='Merge images to the same space'
+        description=(
+            "Merge multiple source images onto destination space. All images are warped to the destination space and "
+            "then added."
+            "\n"
+            "\nTo deal with overlap during merging (e.g. one voxel in destination image is shared with two input "
+            "images), the resulting voxel is divided by the sum of the partial volume of each image."
+            "\n"
+            "\nFor example, if src(x,y,z)=1 is mapped to dest(i,j,k) with a partial volume of 0.5 (because destination "
+            "voxel is bigger), then its value after linear interpolation will be 0.5. To account for partial volume, "
+            "the resulting voxel will be: dest(i,j,k) = 0.5*0.5/0.5 = 0.5."
+            "\n"
+            "\nNow, if two voxels overlap in the destination space, let's say: src(x,y,z)=1 and src2'(x',y',z')=1, "
+            "then the resulting value will be: dest(i,j,k) = (0.5*0.5 + 0.5*0.5) / (0.5+0.5) = 0.5. "
+            "\n"
+            "\nSo this function acts like a weighted average operator, only in destination voxels that share multiple "
+            "source voxels."
```

Comment:
Even though the docstring description is an improvement over the old description... Admittedly, I find these examples to be a bit confusing? 

(Since all of the values in the examples are `0.5`, it's really hard to tell which `0.5` values refer to specific quantities, e.g. partial volume, data, etc.)

I want to try my hand at rewriting this a little to make it a bit more clear.

Reply:
I've taken a stab at this in [c8705ef](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3840/commits/c8705ef960d83cff2cba8308c7d6dfbca3733131), by being more explicit about what quantities are actually being used in the calculation.

Reply:
If the new description looks good, perhaps we should also copy it back down to the docstring of the function itself?

(Though, maybe we're better off keeping a minimal docstring for the function, and having the detailed explanation be in the argparse help instead.)

Reply:
Good idea, you can copy it to the docstring yes.